packages/manager/modules/pci/src/components/project/users/users.html

```diff
@@ -55,13 +63,14 @@
                 }"
             >
                 <span
-                    data-ng-bind=":: 'pci_projects_project_users_status_' + $row.status | translate"
+                    data-ng-bind=":: (('pci_projects_project_users_status_' + $row.status).toLowerCase() | translate)"
```

Comment:
This should be enough no ?
```suggestion
                    data-ng-bind=":: 'pci_projects_project_users_status_' + $row.status.toLowerCase() | translate"
```

Reply:
Fixed!